ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -21,6 +21,13 @@ images:
       as:
       - registry.ci.openshift.org/openshift/release:golang-1.17
   to: hypershift
+- dockerfile_path: Dockerfile.control-plane
+  from: base
+  inputs:
+    openshift_release_golang-1.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.17
+  to: hypershift-control-plane-operator
```

Comment:
We have to exclude this from promotion I think

Reply:
Agree, will fix